sql/moz-fx-data-shared-prod/telemetry_derived/newtab_merino_extract_to_gcs_v1/metadata.yaml

```diff
@@ -15,7 +15,7 @@ scheduling:
   - --source-dataset=telemetry_derived
   - --source-table=newtab_merino_extract_v1
   - --destination-bucket=merino-airflow-data-prodpy
-  - --destination-prefix=newtab-merino-exports
+  - --destination-prefix=newtab-merino-exports/engagement
```

Comment:
I'm adding a directory to distinguish the engagement data from the prior data. This means this PR should ideally go out within a few hours of a PR in Merino, to avoid stale engagement data.